server/src/test/java/org/spine3/server/aggregate/AggregatePartShould.java

```diff
@@ -89,6 +90,11 @@ public void obtain_aggregate_part_constructor() {
         assertNotNull(constructor);
     }
 
+    @Test
+    public void have_TypeInfo() {
+        assertHasPrivateParameterlessCtor(AggregatePart.TypeInfo.class);
```

Comment:
`TypeInfo` is the utility class. In the method name we declare that type information should be obtained via such class. In the body of the method we cover the utility constructor.